configuration/enable-offline-mode.adoc

```diff
@@ -46,14 +46,16 @@ include::modules/enable-offline-mode-roxctl.adoc[leveloffset=+2]
 [id="update-scanner-definitions-in-offline-mode"]
 == Updating Scanner definitions in offline mode
 
-Scanner contains a local vulnerability definitions database. When {product-title} runs in normal mode (connected to the internet), Scanner fetches new vulnerability definitions from the internet and updates its database.
+Scanner maintains a database of vulnerabilities that it retrieves from Central. When {rh-rhacs-first} is running in normal mode, Central retrieves the latest vulnerability data from the internet and passes it on to Scanner.
```

Comment:
```suggestion
Scanner maintains a database of vulnerabilities. When {rh-rhacs-first} runs in normal mode, Central retrieves the latest vulnerability data from the internet and sends the data to Scanner.
```
better to remove personification and "it, which is not searchable or clear
The rest of this section talks about Scanner receiving the information from Central, not retrieving it. Which is it?

Reply:
FWIW - Scanner 'pulls' vuln data from Central, and Central pulls the vuln data from the internet.  Central never 'initiates' sending vuln data to Scanner.  Perhaps:

```suggestion
Scanner maintains a database of vulnerabilities. When {rh-rhacs-first} runs in normal mode, Central retrieves the latest vulnerability data from the internet, and Scanner retrieves vulnerability data from Central.
```